### PR TITLE
Updated to use centralised CI and standard version format for changelog

### PR DESCRIPTION
**Pull request by alourie**
Signed-off-by: Alex Lourie <alex.lourie@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [OPS-158 Standardise changelog format across all repos](https://app.clickup.com/t/6929263/OPS-158)

File: .github/workflows/python-lib-release.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Python Library Release
+
+on: workflow_dispatch
+
+jobs:
+  run:
+    uses: zepben/.github/.github/workflows/python-lib-release.yml@main
+    with:
+      private: true
```
Comment (kgreav) [inline .github/workflows/python-lib-release.yml]: this one is public

**Review by kgreav — CHANGES_REQUESTED** [review]
Also add the LTS/hotfix workflows please

**Review by kgreav — CHANGES_REQUESTED** [review]
you'll need to add the licencecheck to the build and snapshot workflows - sorry missed this before

**Review by kgreav — APPROVED** [review]

**Pull request merged**